desktop/cmp/panel/PanelModel.js

```diff
@@ -111,16 +111,11 @@ export class PanelModel {
 
         apiRemoved(rest.prefName, 'prefName', 'Specify "persistWith" instead.');
 
-        if (!isNil(maxSize) && (maxSize < minSize || maxSize < defaultSize)) {
-            console.error("'maxSize' must be greater than 'minSize' and 'defaultSize'. No 'maxSize' will be set.");
-            maxSize = null;
-        }
-
         this.collapsible = collapsible;
         this.resizable = resizable;
         this.resizeWhileDragging = resizeWhileDragging;
         this.defaultSize = defaultSize;
-        this.minSize = Math.min(minSize, defaultSize);
+        this.minSize = minSize,
```

Comment:
With minSize now placed as an attribute on the cmp in ResizeContainer.js, I think this simplified setting works.